contract_invoice_auto_validate/migrations/18.0.1.0.1/post-migrate.py

```diff
@@ -0,0 +1,6 @@
+# Copyright 2025 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+def migrate(cr, version):
+    cr.execute("UPDATE res_company SET auto_post_contract_invoice=True")
```

Comment:
@sbejaoui I'm not sure this should be part ot the module as it changes behavior per default

Reply:
but for users who had this module in previous versions, the feature was enabled by default. With this migration, unless the value is explicitly set to true, the feature will be disabled for them.